x-pack/plugins/infra/common/alerting/logs/log_threshold/types.ts

```diff
@@ -92,6 +92,58 @@ export const ComparatorToi18nMap = {
   ),
 };
 
+export const ComparatorToi18nSymbolsMap = {
+  [Comparator.GT]: i18n.translate('xpack.infra.logs.alerting.comparator.symbol.gt', {
```

Comment:
@fkanout Why do we need to translate symbols?

Reply:
@fkanout No I don't mean `[Comparator.GT]`. I mean why do you have to translate `>` by using i18n.translate? A simple map like this:

`[Comparator.GT_OR_EQ]: '>'`

would do the trick, no?

Do I miss something here?

Reply:
@mgiota, it will definitely do the trick. I thought about consistency as it's a text and it should be i18n.
Do you want me to do it with the direct map without i18n. e.g. I don't think `>` in another language it could be something else 😄

Reply:
@fkanout Yep I would prefer to get rid of i18n, because it is not needed anymore. As you said `>` will always be `>`, so no need to localize it.

Reply:
Do we support languages written right to left?